src/m-r/RocWb/RocWb.StaticSeries.cs

```diff
@@ -67,18 +63,28 @@ public static IReadOnlyList<RocWbResult> ToRocWb<T>(
 
             prevEma = ema[i];
 
-            // ROC deviation
+            // ROC standard deviation (proper formula)
```

Comment:
You must consult with the referenced canonical source formula, typically a link in the first paragraph of our `/docs/_indicators/RocWb.md` documentation site pages before making assumptions about indicator formulas.  In this case you need to #fetch https://en.wikipedia.org/wiki/Momentum_(technical_analysis) to see the correct formula -- you may have jumped to a conclusion about "deviation" being the same as "standard deviation" here.  If you are not sure, you must stop and ask me; however, you should always initially assume the static series style indicators are correct as they've been hardened over time.